commands/action/nom.js

```diff
@@ -0,0 +1,27 @@
+const { MessageEmbed } = require('discord.js')
+const { nom } = require('../../assets/json/images.json')
```

Comment:
Use the image manager for this one. Code has to be consistent all throughout the repository

To do this:
1. Go to `root/struct/Images.js`
2.  insert this function in alphabetical order
```js
.actionname(){
   return this._action('actionname')
   }
```
`actionname` is a placeholder for the actual action type, for example, you added this nom action, you would replace `actionname` with `nom`. This becomes
```js
.nom(){
   return this._action('nom')
   }
```
Be careful tho, `nom` has to exist on the json file in assets/json/images.json, or it will error. Do the same for the other commands

3. instead of using the gibberish `nom[Math.ceil(Math.random() * nom.length)]}`, you can now access it via `client.images.nom()` <-- this will return a random imgur url.

Check the other action command codes for reference.